src/NetAppFiles/NetAppFiles/BackupPolicy/SetNetAppFilesBackupPolicy.cs

```diff
@@ -0,0 +1,172 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Management.NetApp;
+using System.Globalization;
+using Microsoft.Azure.Commands.NetAppFiles.Helpers;
+using Microsoft.Azure.Management.Monitor.Version2018_09_01.Models;
+using System.Collections.Generic;
+using System;
+
+namespace Microsoft.Azure.Commands.NetAppFiles.BackupPolicy
+{
+    [Cmdlet(
+        "Set",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesBackupPolicy",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesBackupPolicy))]
+    [Alias("Set-AnfBackupPolicy")]
+    public class SetAzureRmNetAppFilesBackupPolicy : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The resource group of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter()]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The location of the resource")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.NetApp/netAppAccounts/backupPolicies")]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ParameterSetName = FieldsParameterSet,
+            HelpMessage = "The name of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceNameCompleter(
+            "Microsoft.NetApp/netAppAccount",
+            nameof(ResourceGroupName))]
+        public string AccountName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the ANF backup policy")]
+        [ValidateNotNullOrEmpty]
+        [Alias("BackupPolicyName")]
+        [ResourceNameCompleter(
+            "Microsoft.NetApp/netAppAccounts/backupPolicies",
+            nameof(ResourceGroupName),
+            nameof(AccountName))]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The property to decide policy is enabled or not")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter Enabled { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Daily backups count to keep")]
+        [ValidateNotNullOrEmpty]
+        public int? DailyBackupsToKeep { get; set; }
```

Comment:
Powershell prefer to singular name. please change DailyBackupsToKeep, MonthlyBackupsToKeep, WeeklyBackupsToKeep, YearlyBackupsToKeep to xxBackupCountToKeep

Reply:
That would conflict with New-NetAppFilesBackupPolicy where those parameters are already defined this way. Does this mean we have to change that was well and wont that be a breaking change?

Reply:
> That would conflict with New-NetAppFilesBackupPolicy where those parameters are already defined this way. Does this mean we have to change that was well and wont that be a breaking change?

You can keep the style for this cmdlet to keep consistency but please do not use plural names for further cmdlets.